### PR TITLE
Include QC failed reads when recreating per lane bam from merged bam

### PR DESCRIPTION
**Pull request by dufeiyu**
Most of our in-house bams do not include qc-failed reads, but some bams do include those, while currently recreate_bam command has include_qc_failed turned off. This bug causes unexpected bam flagstat diff and bam recreation failing. This PR is to fix the bug and recreate per lane bam to its original.

### CODE REVIEW

File: lib/perl/Genome/InstrumentData/AlignmentResult/Command/RecreatePerLaneBam.pm [hunk 1]
```diff
@@ -41,6 +41,12 @@ class Genome::InstrumentData::AlignmentResult::Command::RecreatePerLaneBam {
             is  => 'FilePath',
             doc => 'The path of flagstat file that is used to compare with that of recreated bam',
         },
+        include_qc_failed => {
+			is          => 'Boolean',
+			doc         => 'Include reads that were marked within the bam as having failed QC',
+			is_optional => 1,
+			default     => 0,
+		},
```
Comment (ebelter) [inline lib/perl/Genome/InstrumentData/AlignmentResult/Command/RecreatePerLaneBam.pm]: The indentation is off here.

Comment (dufeiyu) [inline lib/perl/Genome/InstrumentData/AlignmentResult/Command/RecreatePerLaneBam.pm]: Messed up during copy/paste. Fixed.

**Comment by acoffman** [comment]
+1

**Comment by tmooney** [comment]
+1 :newspaper:

**Comment by ebelter** [comment]
+1

**Pull request merged**